### ISSUE TITLE
A11y_Visual Studio Code Web Extensions_Github Pull request_Create Pull request_Keyboard: Visually label is not provided for combo box in 'Merge changes from' section.

### ISSUE THREAD

**Issue by sspatil123** [microsoft/vscode-pull-request-github#4281]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#GH_Visual Studio Code Web Extensions_Nov2022;#Visual Studio Code Web Extensions ;#ChromiumEdge;#DesktopWeb;#A11ySev2;#Win11;#WCAG3.3.2;#Keyboard;

## Environment Details:
Application Name: Visual Studio Code Web Extensions
URL: https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github
Windows Version: Win11
Edge version:107.0.1418.56

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix
2. Open https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github environment and login with git hub account.
3. tab to 'Github Pull request' and press enter.
4. tab to 'Create Pull request' and press enter.
5. Check label provide for combo box is correct or not.

## Actual:
Visually label is not provided for combo box in 'Merge changes from' section.

**Similar issue observed in 'INTO' Section on same screen.**

## Expected:
Visually label should be provided for combo box in 'Merge changes from' section.

## User Impact:
Keyboard users will not get complete information of combo box.

## Attachment:

![Visual Name not provided](https://user-images.githubusercontent.com/85863999/205224701-d5eb6cdb-6804-4612-be74-e2f2c4fbbb3f.png)

**Comment by daviddossett** [comment]
Looks reasonable to me 👍

**Comment by ManishaC1** [comment]
Github tags:#Closed
![Screenshot (445)](https://user-images.githubusercontent.com/105198219/217433825-4ccd81d3-1b7f-4c46-a4c5-c088964d1d37.png)
